bin/flutterflow_cli.dart

```diff
@@ -79,6 +80,10 @@ ArgResults _parseArgs(List<String> args) {
           'download all project code directly into the specified directory, '
           'or the current directory if --dest is not set.',
       defaultsTo: true,
+    )
+    ..addFlag(
+      'as-module',
```

Comment:
Nit: let's make this option negatable, like so users can explicitly set `--no-as-module` if they want. It's not particularly important, but we do it for other boolean flags.

If you do it, please update README.md: `--as-module` -> `--[no-]as-module` (in both places).

Reply:
I thought if the user do not have this flag, then we don't export the code as module. But I think it makes sense to follow the rule for other boolean flags.